modules/tinymce/src/themes/silver/test/ts/phantom/sketchers/SilverMenubarTest.ts

```diff
@@ -109,14 +109,14 @@ UnitTest.asynctest('SilverMenubar Test', (success, failure) => {
                   return s.element('div', {
                     classes: [ arr.has('tox-collection__item') ],
                     children: [
-                      s.element('div', { classes: [ arr.has('tox-collection__item-icon') ] }),
+                      ...hasIcons ? [ s.element('div', { classes: [ arr.has('tox-collection__item-icon') ] }) ] : [],
                       s.element('div', {
                         classes: [ arr.has('tox-collection__item-label') ],
                         html: str.is(hasCaret ? itemText.substring(0, itemText.length - 1) : itemText)
                       })
                     ].concat(hasCaret ? [
                       s.element('div', { classes: [ arr.has('tox-collection__item-caret') ] })
-                    ] : [ ])
+                    ] : hasCheckmark ? [ s.element('div', { classes: [ arr.has('tox-collection__item-icon'), arr.has('tox-collection__item-checkmark') ] }) ] : [])
```

Comment:
This seems odd. Why are we adding the `icon` class, as that's meant to be used for icons?

Reply:
I assume this may have been to try and get the styles to show for the checkmark? If so I'd suggest changing that in the CSS rather than adding a potentially misleading or confusing class (similar to the caret for example).

Reply:
Thanks for bringing this up. I have made changes so that the `checkmark` class is self-sufficient i.e. the `icon` class is no longer required for checkmarks.